committee-steering/governance/ug-governance.md

```diff
@@ -0,0 +1,37 @@
+# Kubernetes User Group (UG)
+
+## Introduction
+The Kubernetes community and ecosystem benefits providing a place for communication and collaboration between users and community members on topics that either do not have deliverables or are outside the scope of the project itself. These topics are not appropriate for forming SIGs, WGs or Committees, which are intended to drive work within the project. User Groups as the venue for this kind of communication and collaboration.
+
+Example User Groups:
+
+- Big data - Topics related to Big Data integrations.
+- Public Cloud - Topics related to public cloud integrations.
+- Machine Learning - Topics related to ML ecosystem integrations.
+- DevOps - Topics related to DevOps tools and deployments.
+
+## UG Goals
+- Facilitate communication between Kubernetes users and contributors on topics pertaining to - the usage of, the extension of and integration with Kubernetes and Kubernetes subprojects.
+
+## UG Anti - Goals
```

Comment:
small nit: can we rename this to `UG Non-Goals`? 

I think it's more consistent with the terminology we use in other places.

Reply:
Those mean different things - :p

Non-Goals typically not intentionally avoided, but outside the scope of the proposal
Anti-Goals are intentionally avoided, and the scope may include intentional avoidance of them

Reply:
Good point